hack/make-rules/test-integration.sh

```diff
@@ -62,7 +62,7 @@ runTests() {
   # TODO: Re-enable race detection when we switch to a thread-safe etcd client
   # KUBE_RACE="-race"
   make -C "${KUBE_ROOT}" test \
-      WHAT="$(kube::test::find_integration_test_dirs ${2-} | paste -sd' ' -)" \
+      WHAT="$(kube::test::find_integration_test_dirs ${2-} | paste -sd' ' -) ${3-} " \
```

Comment:
We prefer `${var:-'default'}` since it's not often that someone wants to pass an null var around as a meaningful value.